docs/development/custom-vectors/rsa-oaep-sha2/generate_rsa_oaep_sha2.py

```diff
@@ -94,11 +94,14 @@ def build_vectors(mgf1alg, hashalg, filename):
             output.append(b"# Encryption:")
             output.append(binascii.hexlify(ct))
 
+    for i, o in enumerate(output):
+        if type(o) != bytes:
+            output[i] = bytes(str(o).encode("utf-8"))
```

Comment:
This isn't the right way to do this. We need to ensure we're putting the same type of objects in the output list in the first place.

Reply:
OK, now simplify the logic to output `str` only.

Reply:
```
would reformat docs/development/custom-vectors/rsa-oaep-sha2/generate_rsa_oaep_sha2.py
Oh no! 💥 💔 💥
1 file would be reformatted, 241 files would be left unchanged.
ERROR: InvocationError for command /home/runner/work/cryptography/cryptography/.tox/flake/bin/black --check . (exited with code 1)
___________________________________ summary ____________________________________
ERROR:   flake: commands failed
Error: Process completed with exit code 1.
```
flake installation unstable?

Reply:
This is saying that `black` needs this file to be reformatted.